lib/iris/tests/integration/concatenate/test_concatenate.py

```diff
@@ -187,6 +188,53 @@ def test_concatenate_along_anon_dim(self):
         result = concatenate([cube1, cube2])
         self.assertEqual(len(result), 2)
 
+    def _get_3d_cube(self):
+        cube = stock.simple_3d()
+        coord = cube.coord('wibble')
+        cube.remove_coord(coord)
+        cube.add_aux_coord(coord, 0)
+        return cube
```

Comment:
This should be moved up further up the class rather than being in the middle of the `def test_...` tests